__docs__/wonder-blocks-clickable/clickable.argtypes.ts

```diff
@@ -28,6 +28,15 @@ export default {
             type: "text",
         },
     },
+    tabIndex: {
+        description: "Set the tabindex attribute on the rendered element.",
+        control: {type: "number", min: -1},
```

Comment:
**praise:** Thanks for restricting this! one minor suggestion would be to also include `max: 1`, so users are aware about not putting any arbitrary number for a11y reasons (like letting the browser follow its natural flow order).